src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -130,14 +130,16 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             }
             else
             {
+                progress?.Report(new MigrationProgress(0, 1, "Skipping cleaning as no dirty files were found...", false));
                 logger.WriteLine(MigrationStrings.Process_SkippingChecking);
             }
 
-            progress?.Report(new MigrationProgress(0, 1, "Create new binding files ...", false));
+            progress?.Report(new MigrationProgress(0, 1, "Creating new binding files ...", false));
             logger.WriteLine(MigrationStrings.Process_ProcessingNewBinding);
 
             var progressAdapter = new FixedStepsProgressToMigrationProgressAdapter(progress);
             await unintrusiveBindingController.BindAsync(oldBinding, progressAdapter, token);
+            progress?.Report(new MigrationProgress(0, 1, "Saving new binding files ...", false));
```

Comment:
To me it feels weird not having this here so I added it but lmk what you think

Reply:
I don't think it's needed, but it doesn't make a big difference.
Does it cause the vertical scroll bar to appear? If so, it means the users will always have to scroll to see the last message, which isn't ideal.